static/app/views/acceptProjectTransfer/index.tsx

```diff
@@ -42,9 +42,14 @@ class AcceptProjectTransfer extends DeprecatedAsyncView<Props, State> {
       },
       success: () => {
         const orgSlug = formData.organization;
-
-        this.props.router.push(normalizeUrl(`/organizations/${orgSlug}/projects/`));
-        addSuccessMessage(t('Project successfully transferred'));
+        const projectSlug = this.state?.transferDetails?.project.slug;
+        const sentryUrl = ConfigStore.get('links').sentryUrl;
+        if (!projectSlug) {
+          window.location.href = `${sentryUrl}/organizations/${orgSlug}/projects/`;
+        } else {
+          window.location.href = `${sentryUrl}/settings/${orgSlug}/projects/${projectSlug}/teams/`;
```

Comment:
Tested this locally and I think this URL structure is wrong, it should have /organizations/org-slug/settings/ - the first part with the org is important for the redirect. So it would be:

```
window.location.href = ${sentryUrl}/organizations/${orgSlug}/settings/projects/${projectSlug}/teams/
```